.gitignore

```diff
@@ -45,3 +45,4 @@ build
 *.tsbuildinfo
 
 size-snapshot.json
+.generated
```

Comment:
🤔  Shouldn't prettier catch these missing newlines?

Reply:
Prettier is not formatting this file from what I can see.